subxt/src/error.rs

```diff
@@ -107,13 +107,14 @@ impl From<DispatchError> for Error {
 /// An RPC error. Since we are generic over the RPC client that is used,
 /// the error is boxed and could be casted.
 #[derive(Debug, thiserror::Error)]
-#[error("RPC error")]
 pub enum RpcError {
     // Dev note: We need the error to be safely sent between threads
     // for `subscribe_to_block_headers_filling_in_gaps` and friends.
     /// Error related to the RPC client.
+    #[error("{0}")]
```

Comment:
I don't remember if all these errors actually states that it's a `RpcError`.

Might be better to add  `#[error("RPC error: {0}")]` or something?

Reply:
Since the error is very generic and could be anything, I'd agree with soomething like the above :)

Reply:
done